packages/framework-integration-tests/src/sensor/health/application-health-indicator.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  BoosterConfig,
+  HealthIndicatorResult,
+  HealthIndicatorMetadata,
+  HealthStatus,
+} from '@boostercloud/framework-types'
+import { HealthEndpoint } from '@boostercloud/framework-core'
+
+@HealthEndpoint({
+  id: 'myApplication',
+  name: 'my-application',
+  enabled: true,
+  details: true,
+  showChildren: true,
+})
+export class ApplicationHealthIndicator {
+  public async health(
+    config: BoosterConfig,
+    healthIndicatorMetadata: HealthIndicatorMetadata
+  ): Promise<HealthIndicatorResult> {
+    return {
+      status: HealthStatus.UP,
+    } as HealthIndicatorResult
+  }
```

Comment:
Question about parent -> children relationship: 
~When your child doesn't stop crying and you have tried everything~ errr, Is the parent sensor required to also return information about their children? Or is that already managed automatically?

Reply:
it's managed automatically. See the code you suggested the refactor about children process: https://github.com/boostercloud/booster/pull/1421/files#diff-f5101051c59853404201e54e7e21c9aee4585ace4f8077257a0b166d772f5a40R45